content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Secure Supply Chain: Verifying image signatures"
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+After these last releases Kubewarden now has support for verifying the integrity and authenticity of artefacts within
+Kubewarden using the Sigstore project. In this post, we shall focus on verifying container image signatures using the
+new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy
+
+If you don't know how Sigstore works, we recommend reading our
+previous [post](https://www.kubewarden.io/blog/2022/04/securing-kubewarden-policies/)
+
+## Verify Image Signatures Policy
+
+This policy validates Pods by checking their container images for signatures (that is, containers, init containers and 
+ephemeral containers in the pod). If all signature validations pass or there is no container image that matches the image
+name query, the Pod will be accepted.
```

Comment:
I personally prefer something like that, the second sentence is not super clear to me

```suggestion
ephemeral containers in the pod).

The policy can inspect all the container images defined inside of a Pod or it can just analyze the ones that
are matching a pattern provided by the user.
```